lib/improver/tests/ensemble_calibration/ensemble_calibration/test_ApplyCoefficientsForEnsembleCalibration.py

```diff
@@ -90,8 +103,10 @@ def setUp(self):
         """Test up test cubes."""
         data = np.ones([2, 2], dtype=np.float32)
         self.current_forecast = set_up_variable_cube(data)
-        self.coefficients_cube = (
-            set_up_variable_cube(data, name="emos_coefficients"))
+        coeff_names = ["gamma", "delta", "alpha", "beta"]
```

Comment:
Typo above: `"""Set up test cubes"""`

Reply:
I've corrected this typo, and another identical typo as well.